src/integrationTest/kotlin/com/sourcegraph/cody/chat/ChatTest.kt

```diff
@@ -0,0 +1,103 @@
+package com.sourcegraph.cody.chat
+
+import com.intellij.testFramework.runInEdtAndGet
+import com.sourcegraph.cody.agent.CodyAgentService
+import com.sourcegraph.cody.agent.WebviewMessage
+import com.sourcegraph.cody.agent.protocol.GetRepoIdsParam
+import com.sourcegraph.cody.agent.protocol.Repo
+import com.sourcegraph.cody.chat.ui.ContextFileActionLink
+import com.sourcegraph.cody.util.CodyIntegrationTextFixture
+import com.sourcegraph.cody.util.CustomJunitClassRunner
+import com.sourcegraph.cody.util.TestingCredentials
+import java.awt.Component
+import java.awt.Container
+import java.util.concurrent.CompletableFuture
+import java.util.concurrent.TimeUnit
+import junit.framework.TestCase
+import org.awaitility.kotlin.await
+import org.awaitility.kotlin.until
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(CustomJunitClassRunner::class)
+class ChatTest : CodyIntegrationTextFixture() {
+  override fun myCredentials() = TestingCredentials.enterprise
+
+  @Test
+  fun testRemoteContextFileItems() {
+
+    val result = CompletableFuture<List<Repo>>()
+    CodyAgentService.withAgent(project) { agent ->
+      try {
+        val codebaseNames = listOf("github.com/sourcegraph/cody")
+        val param = GetRepoIdsParam(codebaseNames, codebaseNames.size)
+        val repos = agent.server.getRepoIds(param).get()
+        result.complete(repos?.repos ?: emptyList())
+      } catch (e: Exception) {
+        result.complete(emptyList())
```

Comment:
Maybe we can replace it with assert with exception message logged out?
This way if it will fail we will also know why.

Reply:
done